src/pathToVictory/services/pathToVictory.service.ts

```diff
@@ -398,28 +405,67 @@ export class PathToVictoryService extends createPrismaBase(
         })
       }
 
-      let p2vStatus: P2VStatus = P2VStatus.waiting
-      if (
-        pathToVictoryResponse?.counts?.projectedTurnout &&
+      const p2vStatus: P2VStatus =
+        p2vStatusOverride ??
+        (pathToVictoryResponse?.counts?.projectedTurnout &&
         pathToVictoryResponse.counts.projectedTurnout > 0
-      ) {
-        p2vStatus = P2VStatus.complete
+          ? P2VStatus.complete
+          : P2VStatus.waiting)
+
+      const p2vData = (p2v.data || {}) as Record<string, unknown>
```

Comment:
Pretty sure we know what the `p2v.data` looks like, _somewhere_.. please don't use `unknown` when we _can_ type something out.